docs/usage.rst

```diff
@@ -26,44 +26,103 @@ configured storage nodes.  The client nodes must all share the same
 storage nodes.  The nodes must be running.
 
 
-Running a Magic Folder process
-------------------------------
+Creating a Magic Folder Daemon Configuration
+--------------------------------------------
+
+The "Magic Folder Daemon" is a long-running process that accomplishes
+the task of synchronizing all the "magic folders" that are configured
+in it.
+
+A Magic Folder Daemon needs to have some configuration to start. There
+are two ways to create this: from scratch; or from a "legacy"
+magic-folder (when ``magic-folder`` was a sub-command of ``tahoe`` in
+version 1.14.0 and earlier).
+
+No matter the method used, there are some pieces of information
+required and two decisions to be made. You need to know:
+
+- the ``node-directory`` of the Tahoe-LAFS client this magic folder daemon shall use.
+
+You need to decide:
+
+- The network endpoint the API will listen on. This is how CLI commands and front-ends communciate to the magic-folder daemon and is expressed as a Twisted "server endpoint string". For example, ``tcp:4321`` to listen globally on port 4321. You may also use Unix Domain sockets on OSes which support that; for example ``unix:/var/run/magic-folder/api-socket``.
+- If you wish to specify where to store the configuration. By default
+  it will be in an appropriate location for your OS (like
+  ``~/.config/magic-folder`` on Debian).
+
+To create a magic folder daemon configuration from scratch:
+
+.. code-block:: console
+
+   $ magic-folder init --config ./foo --listen-endpoint tcp:4321:interface=localhost --node-directory ./tahoe-client
+
+This will store configuration in ``~/.config/magic-folder/*`` or
+similar; listen on ``localhost`` port ``4321`` for API commands; and
+talk to the Tahoe-LAFS client in ``./tahoe-client`` (which must itself
+be running).
 
-If you are running a Tahoe-LAFS client off a Tahoe-LAFS node directory
-``alice``, like so:
+To create a magic folder daemon configuration by migrating an existing
+Tahoe-LAFS-based magic-folder, do:
 
 .. code-block:: console
 
-   $ tahoe --node-directory=alice run
+   $ magic-folder migrate --config ./foo --listen-endpoint tcp:4321:interface=localhost --node-directory ./tahoe-client --author alice
```

Comment:
`tahoe migrate` is creating a brand-new configuration somewhere (so there can be no name conflicts). If we added the ability for `migrate` to move "some" folders over to an existing new-style daemon I guess we'd have to decide what to do if there was already a folder (but each folder has its own author).

So, since each folder has its own author, if you identify them all with "meejah" (for example) then there are multiple different key-pairs each called "meejah". I've been *trying* to use "device" as well to highlight the fact that the "author" here is per-magic-folder-client and not something with wider scope.

Reply:
Those seem like good reasons to present the information but I'm not sure if they argue for the information to be presented here.

On the other hand, we have no overall information architecture so I guess here is as good as anywhere.

Reply:
uh ignore that, this was supposed to go on some other thread